web/packages/teleport/src/Integrations/Enroll/common.tsx

```diff
@@ -57,3 +64,84 @@ export const NoCodeIntegrationDescription = () => (
     </Text>
   </Box>
 );
+
+export const MachineIDIntegrationSection = () => {
+  interface tile {
+    title: string
+    link: string
+    icon: JSX.Element
+  }
+  const tiles: tile[] = [
+    // TODO: Emit an event or pass these through a /r/ url to redirect to a
+    // URL with tracking.
+    // Can we emit events in OSS and have them go nowhere?? Is it best to use
+    // a /r/ url and have support on both Enterprise and OSS.
+    {
+      title: 'GitHub Actions',
+      link: 'https://goteleport.com/docs/machine-id/guides/github-actions/',
```

Comment:
Keeping track of some of my owns thoughts:

Do we want to link these through a `/r/` redirect and then attach UTM details ? Or do we want to not do that, and emit an event when on Enterprise/Cloud ? Can I emit an event in OSS and just have it sinkholed or do I need to DI in the emitter in `e` and push in a nop implementation in OSS.